src/tests/Interop/COM/NativeClients/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_CLRTestRunFile>COMClient$(MSBuildProjectName)</_CLRTestRunFile>
```

Comment:
LGTM overall, thanks for the beautiful cleanup. One small nit I have is that I believe the standard practice in our CLRTest.*.targets scripts is to use properties starting with an underscore for internal variables only used within the build harness itself. As you're making <code>_CLRTestRunFile</code> "public" in the sense that it's being set by certain groups of tests, I think we should remove the initial underscore and rename it to just <code>CLRTestRunFile</code>. I don't think you need to do that as part of this change as the mechanical rename would clutter and obscure the semantic changes but it would be nice to do as a follow-up cleanup.